packages/stack/src/types.tsx

```diff
@@ -78,8 +78,19 @@ export type GestureDirection =
   | 'vertical'
   | 'vertical-inverted';
 
+export type StackAnimationName =
+  | 'default'
+  | 'fade'
+  | 'fade_from_bottom'
+  | 'none'
+  | 'reveal_from_bottom'
+  | 'scale_from_center'
+  | 'slide_from_bottom'
+  | 'slide_from_right'
+  | 'slide_from_left';
```

Comment:
@satya164 I've removed the 'bottom_sheet' cause it's the same as 'slide_from_bottom' - diffs only in few miliseconds